sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -289,7 +290,46 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       }
       checkAnswer(
         sql("SELECT employeeID, employeeName FROM part_table WHERE c = '2' AND d = '1'"),
-        sql("SELECT * FROM non_part_table").collect())
+        sql("SELECT * FROM non_part_table"))
+
+      // check for case insensitive property of partition column name in load command.
+      withSQLConf(SQLConf.CASE_SENSITIVE.key -> "false") {
+        withInputFile { f =>
+          sql(s"""$loadQuery INPATH "${f.toURI}" INTO TABLE part_table PARTITION(C="1", D="2")""")
+        }
+        checkAnswer(
+          sql("SELECT employeeID, employeeName FROM part_table WHERE c = '2' AND d = '1'"),
+          sql("SELECT * FROM non_part_table").collect())
+      }
+    }
+  }
+
+  test("SPARK-28084 case insensitive names of static partitioning in INSERT commands") {
+    // check for case insensitive property of partition column name in insert command.
```

Comment:
nit: probably, we don't need this comment. (I think the test title already explains about it)